docs/source/configuration/environmentvariables.md

```diff
@@ -252,3 +252,10 @@ In the frontend, you can access this variable in your code with the following.
 ```shell
 window.env.RAZZLE_MY_VARIABLE
 ```
+## Related content
+
+- {doc}`index`
+- {doc}`volto-config-js`
+- {doc}`settings-reference`
+- {doc}`multilingual`
```

Comment:
I don't see why environment variables are related to a couple of these documents. Can you explain why you added them? Otherwise let's remove them.

In `index.md`, there's no content to read, and `settings-reference` is for programmatically configuring Volto with no mention of environment variable.

```suggestion
- {doc}`volto-config-js`
- {doc}`multilingual`
```